giraffe/src/components/SizedPlot.tsx

```diff
@@ -87,10 +89,23 @@ export const SizedPlot: FunctionComponent<SizedPlotProps> = ({
     forceUpdate()
   }, [env])
 
+  const defaultSpec = env.getSpec(0)
+
+  const valueX = env.xScale.invert(hoverEvent.x)
+  let clampedValueX = NaN
+  if (
+    valueX &&
+    (defaultSpec.type === SpecTypes.Band || defaultSpec.type === SpecTypes.Line)
+  ) {
+    const timestamps = defaultSpec?.lineData[0]?.xs ?? []
```

Comment:
The layer config will have "x" and "y" properties which are column keys, and of which one of them will be "_time" (usually). Shouldn't the timestamps be based finding out which dimension is "_time"?

Not a deal-breaker since this is only needed to find a clamped value. And currently annotations are supported only when x-axis is time.

Reply:
this should work on any rounded numerical value, not just timestamps. I'm not sure what is better - explicitly looking for _time, since that what this seems the most relevant to, or using any x axis value, since that's how giraffe handles it?

Reply:
Or find clamped in both dimensions and let the consumer decide if they need it? i dunno. i'm okay with just one dimension for now.